config/app.default.php

```diff
@@ -263,9 +267,12 @@
              * the following line and set the port accordingly
              */
             //'port' => 'non_standard_port_number',
-            'username' => 'my_app',
-            'password' => 'secret',
-            'database' => 'my_app',
+            /*
+             * It is recommended to set these via your environment or app_local.php
```

Comment:
Is it "via" or "through" in English?

Reply:
"via" is also valid but "through" might help non-native English speaker understand better.

Reply:
I would go with 'through'.

Reply:
i prefer via because it is less ambigious in my mind but I changed it nevertheless.